component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -109,9 +109,9 @@ public JWTValidator(boolean preventTokenReuse, String validAudience, int rejectB
      *
      * @param signedJWT Validate the token
      * @return true if the jwt is valid.
-     * @throws IdentityOAuth2Exception
+     * @throws OAuthClientAuthnException
      */
-    public boolean isValidAssertion(SignedJWT signedJWT) throws OAuthClientAuthnException {
+    public boolean isValidAssertion(SignedJWT signedJWT, boolean isBackchannelCall) throws OAuthClientAuthnException {
```

Comment:
We can't change a public method signature. We need to keep the old definition as it is and define as  deprecated and call the new method.

Reply:
Ack. Readded the old method and deprecated it.